src/store/normalize-data.test.js

```diff
@@ -46,7 +46,7 @@ describe('normalizeData', () => {
     });
     expect(normalizeData(data).pipeline.active).toBe(undefined);
   });
-
+  /* eslint-disable id-length,camelcase */
```

Comment:
it seems like we only need to disable the `camelcase` rule here - I wonder what is the reason behind disabling `id-length` here as well?

Reply:
good spot, not required